src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -68,11 +73,17 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
 
         await this._checkoutValidator.validate(state.checkout.getCheckout(), options);
 
-        state = await this._store.dispatch(
-            this._paymentMethodActionCreator.loadPaymentMethod(paymentId, options)
-        );
+        state = await this._loadPaymentMethod(gatewayId, methodId, options);
```

Comment:
I would change the name because in a glance I feel a little confused,  Are we working with the state or with the payment method?
```suggestion
        state = await this._loadState(gatewayId, methodId, options);
```

Reply:
Even if we are returning the state, the purpose of this method is to reload the payment method, that's why I decided to call it that way